om-2.0.rdf

```diff
@@ -28867,6 +28881,15 @@
     <om:symbol>C</om:symbol>
   </owl:Class>
 
+  <!-- Particle Concentration Upper Ontology -->
+
+  <owl:Class rdf:about="&om;ParticleConcentration">

```

Comment:
particle number concentration for clarity (because there are others like particle mass concentration or particle volume concentration)

Reply:
Changed to Particle Number Concentration Upper Ontology